packages-ts/starknet/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@chainlink/starknet",
+  "version": "1.0.0",
+  "description": "Starknet test utilities",
+  "keywords": [
+    "typescript",
+    "cli"
+  ],
+  "main": "./dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*",
+    "!dist/**/*.test.js"
+  ],
+  "scripts": {
+    "format": "yarn prettier --write ./src",
+    "format:check": "yarn prettier --check ./src",
```

Comment:
Check out other packages and their definitions. The test script definition is missing.